### PR TITLE
Added dashboard page. This allows you to view your subscribed rsos ev…

### PR DESCRIPTION
**Pull request by TylerGauntlett**
…ents, make new events, make rsos and join/leave rsos

### CODE REVIEW

File: src/app/components/dashboard/dashboardView.html [hunk 1]
```diff
@@ -1,3 +1,289 @@
...
+                    <i class="fa fa-eye" aria-hidden="true"></i> View
+                  </button>
+                  <button class="btn btn-danger">
+                    <i class="fa fa-close" aria-hidden="true"></i> Leave
+                  </button>
+                </div>
+              </td>
+            </tr>
+            <tr>
+              <td>Face Painting</td>
```
Comment (Tresky) [inline src/app/components/dashboard/dashboardView.html]: Is this is hardcoded event data?

File: src/app/components/dashboard/dashboardView.html [hunk 2]
```diff
@@ -1,3 +1,289 @@
...
+            <div class="panel-body">
+              <div class="col-md-6">
+                <div class="form-group">
+                  <label for="rso-name">RSO Name:</label>
+                  <input type="text" class="form-control" id="rso-name">
+                </div>
+              </div>
+              <div class="col-md-6">
+                <div class="form-group">
+                  <label for="rso-lead-email">RSO Leader Email:</label>
```
Comment (Tresky) [inline src/app/components/dashboard/dashboardView.html]: Let's not have an RSO leader email, but instead just have the four other emails. The RSO leader will be assumed to be the person who filled out the form.

**Review by Tresky — CHANGES_REQUESTED** [review]

File: src/app/components/dashboard/dashboardView.html (hunk 1 above)
Comment (TylerGauntlett) [reply src/app/components/dashboard/dashboardView.html]: Yes all hard coded.

File: src/app/components/dashboard/dashboardView.html (hunk 2 above)
Comment (TylerGauntlett) [reply src/app/components/dashboard/dashboardView.html]: Rereading the docs, i think you need 5 people PLUS you. Ill double check though.

Comment (TylerGauntlett) [reply src/app/components/dashboard/dashboardView.html]: `A new RSO can be created with at least 5 other students with the same email domain, e.g.@knights.ucf.edu; and one of them should be assigned as an administrator.` from doc

File: src/app/components/dashboard/dashboardView.html (hunk 1 above)
Comment (Tresky) [reply src/app/components/dashboard/dashboardView.html]: I'm assuming this is not permanent. lol

File: src/app/components/dashboard/dashboardView.html (hunk 2 above)
Comment (Tresky) [reply src/app/components/dashboard/dashboardView.html]: So, we need 6 people? If so, let's still assume that the person who is creating it is the new admin. Maybe have as part of the UI that says: 'RSO Admin' and it has the current user's name (or email) next to it in a disabled (can use `ng-disabled=true` attribute for this) text box? 

Just an idea, can play around with it.

File: src/app/components/dashboard/dashboardView.html (hunk 1 above)
Comment (TylerGauntlett) [reply src/app/components/dashboard/dashboardView.html]: Its all just html right now. Made the views as quickly as possible so Koha always had something to work on. Will add functionality this weekend.

File: src/app/components/dashboard/dashboardView.html (hunk 2 above)
Comment (TylerGauntlett) [reply src/app/components/dashboard/dashboardView.html]: Did exactly that

**Review by Tresky — APPROVED** [review]

**Pull request merged**